public/script.js

```diff
@@ -0,0 +1,16 @@
+$(document).ready(function () {
+  $("form").submit(event => {
+    event.preventDefault();
+    axios.get("http://localhost:3000/randomQuote").then(response => {
```

Comment:
this was changed from a `.post` to `.get` and then routed back to the random quotes path.